src/transformers/generation/utils.py

```diff
@@ -878,6 +878,7 @@ def _get_logits_processor(
                 SuppressTokensAtBeginLogitsProcessor(generation_config.begin_suppress_tokens, begin_index)
             )
         if generation_config.forced_decoder_ids is not None:
+            forced_decoder_ids = [[input_ids_seq_length + i[0] - 1, i[1]] for i in forced_decoder_ids]
             processors.append(ForceTokensLogitsProcessor(generation_config.forced_decoder_ids))
```

Comment:
this should either be used or not included? 
 ```suggestion
            processors.append(ForceTokensLogitsProcessor(generation_config.forced_decoder_ids))
```